docs/news.rst

```diff
@@ -3,6 +3,158 @@
 Release notes
 =============
 
+1.1.0
+-----
+
+This 1.1 release brings a lot of interesting features and bug fixes:
+
+- Scrapy 1.1 has basic Python 3 support (requires Twisted >= 15.5). See
+  :ref:`news_basicpy3` for more details and some limitations.
+- Hot new features:
+
+  - ItemLoaders now support nested loaders (:issue:`1467`).
+  - ``FormRequest.from_response`` improvements (:issue:`1382`, :issue:`1137`).
+  - Added setting :setting:`AUTOTHROTTLE_TARGET_CONCURRENCY` and improved
+    AutoThrottle docs (:issue:`1324`).
+  - Added ``response.text`` to get body as unicode (:issue:`1730`).
+  - Anonymous S3 connections (:issue:`1358`).
+  - Deferreds in downloader middlewares (:issue:`1473`). This enables better
+    robots.txt handling (:issue:`1471`).
+  - HTTP cache improvements (:issue:`1151`).
+
+- These bug fixes may require your attention:
+
+  - Don't retry bad requests (HTTP 400) (:issue:`1289`).
+  - Fix shell files argument handling (:issue:`1710`, :issue:`1550`).
+  - Fixes on robots.txt handling (:issue:`1783`).
+  - Exporters work on unicode (:issue:`1080`).
+  - Fix xmliter to accept nodenames with dots (:issue:`1533`).
```

Comment:
These bug fixes require user attention because they can break user code. It'd be nice to explain how they can break it, and what to do.